configuration/screensaver.md

```diff
@@ -1,28 +1,34 @@
 # Setting the screen saver/screen blanking
 
-## On the Console
+## On the console
 
-If you are using the Raspberry Pi solely on the console (no desktop GUI), you need to set the console blanking. The current setting, in seconds, can be displayed using
+By default, when running without a graphical desktop, Raspberry Pi OS will blank the screen after 10 minutes without user input: either mouse movement or keyboard input.
+
+The current setting, in seconds, can be displayed using:
 ```
 cat /sys/module/kernel/parameters/consoleblank
 ```
 
-Here, `consoleblank` is a kernel parameter. In order to be permanently set, it needs to be defined on the kernel command line.
+Here, `consoleblank` is a kernel parameter. In order to be permanently set, it needs to be defined on the kernel command line. To edit the kernel command line:
 
 ```
 sudo nano /boot/cmdline.txt
 ```
 
-Add `consoleblank=0` to turn screen blanking off completely, or edit it to set the number of seconds of inactivity before the console will blank. Note the kernel command line must be a single line of text.
+The file `/boot/cmdline.txt` contains a single line of text. Add `consoleblank=n` to have the console blank after `n` seconds of inactivity, for example `consoleblank=300` for 300 seconds = 5 minutes. To disable screen blanking, set `consoleblank=0`. Make sure that you add your `consoleblank` option to the single line of text already in the `cmdline.txt` file.
+
+You can also use the `raspi-config` tool to disable screen blanking. Note that the screen blanking setting in `raspi-config` also controls screen blanking when the graphical desktop is running.
+
+## On the Raspberry Pi desktop
 
-## On the Raspberry Pi Desktop
+By default, Raspberry Pi OS will blank the graphical desktop after 10 minutes without user input. You can disable this by changing the 'Screen Blanking' option on the Raspberry Pi Configuration tool, which is available on the Preferences menu. Note that the 'Screen Blanking' option also controls screen blanking when the graphical desktop is not running.
 
-By default, the Raspberry Pi Desktop does not have any easy-to-use screensaver software installed, although the screensaver is enabled. Firstly, you should install the X Windows screensaver application.
+There is also a graphical screen saver available, which can be installed as follows:
```

Comment:
Looking at e.g. https://www.google.com/search?q=screensaver it probably makes sense to leave "screensaver" as a single word?

Reply:
I was going by lexico.com, which is the free version of the Oxford English Dictionary online. It only has an entry for screen saver as two separate words. The page also predominently used the "two words" version before my edits.

Reply:
Done.